### DISCUSSION TITLE
predictor.serialize()

### DISCUSSION THREAD

**Discussion by pratikgehlott** [awslabs/gluonts#1689] [Q&A]
predictor.serialize() requires us to pass path as the input, but whenever I pass the input as predictor.serialize(os.getcwd()), it gives error, but when I do predictor.serialize(Path("/tmp/")) it does not give error, but I need to use first method, what should I do?

Any comments @lostella , @mbohlkeschneider ?

**Comment by mbohlkeschneider** [comment]
Doesn't `predictor.serialize(Path(os.getcwd()))` work?

**Reply by pratikgehlott** [reply]
yes, thanks alot!!

**Reply by pratikgehlott** [reply]
`WARNING:root:Serializing RepresentableBlockPredictor instances does not save the prediction network structure in a backwards-compatible manner. Be careful not to use this method in production.`

I also get this warning, should I ignore it?
